azurerm/utils/utils.go

```diff
@@ -19,3 +19,21 @@ func Float(input float64) *float64 {
 func String(input string) *string {
 	return &input
 }
+
+func SliceContainsString(haystack []interface{}, needle string) bool {
+	for _, s := range haystack {
+		if s == needle {
+			return true
+		}
+	}
+	return false
+}
+
+func StringSliceContainsStringSlice(leftSide, rightSide []interface{}) bool {
```

Comment:
I don't see this used anywhere?

Reply:
You are right, should I remove it again?

Reply:
👍 pushed a commit to remove this (I hope you don't mind!)